pupil_src/shared_modules/pupil_detector_plugins/detector_base_plugin.py

```diff
@@ -112,7 +115,7 @@ def recent_events(self, event):
             return
 
         # Detect if resolution changed
-        self._update_frame_size_if_changed(new_frame_size=(frame.width, frame.height))
+        self.__update_frame_size_if_changed(new_frame_size=(frame.width, frame.height))
 
         # Get results of previous detectors
         # TODO: This is currently used by Pye3D to get the results of the 2D detection
```

Comment:
@romanroibu What do you think? Should we move this into the pye3d detector plugin? Also, instead of the `event` key, it might be sensible to have a detector-specific key, maybe based on the `identifier`?